tests/qasm/test_qasm_to_mpqp.py

```diff
@@ -30,8 +30,8 @@
             [
                 H(0),
                 CNOT(0, 1),
-                BasisMeasure([0]),
-                BasisMeasure([1]),
+                BasisMeasure([0], [0]),
```

Comment:
wrong ?

Reply:
in contrary with "measure q -> c;" -> `BasisMeasure()`